### ISSUE TITLE
Missing Mitre ID's

### ISSUE THREAD

**Issue by timukas** [MythicAgents/Athena#67]
Latest Athena agent on latest Mythic is missing some Mitre IDs:

![image](https://github.com/MythicAgents/Athena/assets/1915618/bffd0b17-7566-4465-928f-3416e2a2e60b)

**Comment by checkymander** [comment]
Yeah easy fix that I've just been too lazy to do lol. I'll remove those items Monday.

Thanks!
